### PR TITLE
DOC-8724 -- 2.8.6 Release Notes (Android-only)

### PR DESCRIPTION
**Pull request by ibsoln**
https://issues.couchbase.com/browse/DOC-8724

### CODE REVIEW

File: modules/android/pages/_partials/pn-issues-list.adoc [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
+// tag::issues-2-8-6[]
+// ==== Deprecated
+// tag::deprecated-2-8-6[]
+// end::deprecated-2-8-6[]
+
+==== Enhancements
+// tag::enhancements-2-8-6[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=enhancements-2-8-6]
+
+* https://issues.couchbase.com/browse/CBL-1979[CBL-1979] -- Android v30 compatibility
```
Comment (djpongh) [inline modules/android/pages/_partials/pn-issues-list.adoc]: This was inadvertently tagged for this release. Only CBL-2001 should be here

File: modules/android/pages/_partials/pn-issues-list.adoc [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+// end::enhancements-2-8-6[]
+
+
+==== Fixed
+// tag::fixed-2-8-6[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=fixed-2-8-6]
+
+* https://issues.couchbase.com/browse/CBL-1984[CBL-1984] -- Support domain specific server authentication
+* https://issues.couchbase.com/browse/CBL-2037[CBL-2037] -- Core pool size is too small
+* https://issues.couchbase.com/browse/CBL-2089[CBL-2089] -- Replication failed to establish to a sync gateway configured with a system-trusted CA cert
```
Comment (djpongh) [inline modules/android/pages/_partials/pn-issues-list.adoc]: This turned out to be an internal setup error, fixed by IT. Do not include.

**Review by djpongh — CHANGES_REQUESTED** [review]

File: modules/android/pages/_partials/pn-issues-list.adoc (hunk 1 above)
Comment (amarantha-k) [reply modules/android/pages/_partials/pn-issues-list.adoc]: Removed

File: modules/android/pages/_partials/pn-issues-list.adoc (hunk 2 above)
Comment (amarantha-k) [reply modules/android/pages/_partials/pn-issues-list.adoc]: Deleted CBL-2089

**Review by amarantha-k — APPROVED** [review]
lgtm

**Review by djpongh — APPROVED** [review]

**Pull request merged**